pascals-triangle.json

```diff
@@ -0,0 +1,47 @@
+{
+    "#": [
+      "Expectations are represented here as an array of arrays.",
+      "How you represent this idiomatically in your language is up to you.",
+      "An expectation of -1 indicates some sort of failure should occur"
+    ],
+    "rows": {
+      "description": "Given a count, return a collection of that many rows of pascal's triangle",
+      "cases": [
+        {
+            "description": "single row",
+            "count": 1,
+            "expected": [[1]]
+        },
+        {
+            "description": "two rows",
+            "count": 2,
+            "expected": [[1], [1, 1]]
+        },
+        {
+            "description": "three rows",
+            "count": 3,
+            "expected": [[1], [1, 1], [1, 2, 1]]
+        },
+        {
+            "description": "negative rows",
+            "count": -1,
+            "expected": -1
+        },
+        {
+            "description": "no rows",
+            "count": null,
+            "expected": -1
+        }
+      ]
+    },
+    "last_row": {
+      "description": "Returns the last row of the pascal's triangle",
```

Comment:
If we expect `rows` to return a list/array, then wouldn't this already be built in to the language? 

```ruby
r = PascalsTriangle.rows(4)
r[3]
#=> [1, 3, 3, 1]
```

Maybe there's no point to the `last_rows` function, if all it really does is alias built-in array indexing.